### ISSUE TITLE
IDIsp004 should warn here

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#163]
```cs
    public static class C
    {
        public static Stream Stream => File.OpenRead(string.Empty);

        public static long? M() => ↓Stream?.Length;
    }
```

**Comment by jeremyVignelles** [comment]
Not sure that's a correct example. When you have a property, the classic expectation is that the getter doesn't create anything, it only "gets" the value.

**Comment by JohanLarsson** [comment]
In the example above the getter returns a created disposable that is then not disposed in the method.
Agreed a property returning an instance that the caller is supposed to dispose is nasty.

**Comment by jeremyVignelles** [comment]
Yes, but it's doable with the IDisposable annotations ;)
